clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/HealthState.groovy

```diff
@@ -2,5 +2,5 @@
 package com.netflix.spinnaker.clouddriver.model
 
 enum HealthState {
-  Up, Down, Unknown, Starting, OutOfService
+  Up, Down, Unknown, Starting, OutOfService, Succeeded, Failed
```

Comment:
You might want to know if the container running some batch processing task succeeded or failed. In the case where your management system is set to always restart containers on failure (for something like a web server), you'll never see this state. But if your containers are configured to run a task, and exit gracefully, you'll want to know what happened to them. 

The difference is:

`up` signifies the container (or vm) is running properly and passing health checks, whereas `succeeded` tells you the container is no longer running, but has exited successfully (i.e. no signals thrown, exit code 0, etc...).

`down` signifies the container is failing health checks, and should probably be restarted, whereas `failed` tells you the container exited, but not successfully.